pkg/updater/resultstore/resultstore.go

```diff
@@ -521,7 +657,7 @@ func mapStatusToCellResult(testCase *resultstorepb.TestCase) statuspb.TestStatus
 
 // processTestResults iterates through a list of test results and adds them to
 // a map of groupedcells based on the method name produced
-func processTestResults(tg *config.TestGroup, testResults []*resultstorepb.Test, cell updater.Cell, targetID string, sar *singleActionResult, testMethodLimit int, groupedCells map[string][]updater.Cell) {
+func processTestResults(tg *config.TestGroup, groupedCells map[string][]updater.Cell, testResults []*resultstorepb.Test, sar *singleActionResult, cell updater.Cell, targetID string, testMethodLimit int) {
```

Comment:
nit: Any reason this is switching the args around?

Reply:
Just personal preference; I prefer having the lists, map and struct in front of primitive args 😸